lib/inc/facter/facts/fact.hpp

```diff
@@ -434,11 +434,15 @@ namespace facter { namespace facts {
         /**
          * The fact for virtualization hypervisor.
          */
-        constexpr static char const* virtualization = "virtual";
+        constexpr static char const* virt = "virtual";
```

Comment:
Or maybe we leave this one back to how it is, and come up with a better name for the fact below. It might be weird to rename this fact and bring in a new fact with the same name but different information.